sdk/appconfiguration/Azure.Data.AppConfiguration/tests/ConfigurationLiveTests.cs

```diff
@@ -151,24 +151,71 @@ public async Task DeleteSettingWithLabel()
         }
 
         [Test]
-        public async Task DeleteSettingWithETag()
+        public async Task DeleteSettingLocked()
```

Comment:
How consistent do we want to be about saying this is a "read-only" flag that we're setting and not calling it 'lock/unlock'?

Reply:
Fixed - thanks!